src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.ecommercee.domain;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+/**
+ * @author Dawid Nogacz on 21.08.2019
+ */
+
+@Entity(name = "products")
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "price")
+    private BigDecimal price;
+
+    @ManyToOne
+    @JoinColumn(name = "group_id", nullable = false)
+    @JsonBackReference
+    private Group groupId;

```

Comment:
pole powinno się nazywać group

Reply:
https://github.com/interceptor2048/project-jdp-1908-03/blob/master/specs/product.spec.json

Według specyfikacji groupID ;)

Reply:
chyba specyfikacja dotyczy kolumny w bazie, w javie chyba można nazwać jak chcemy

Reply:
Dałem to groupID w DTO więc będzie myślę dobrze zwracać teraz :)